fiasco/util/setup_db.py

```diff
@@ -28,7 +28,7 @@
 ]
 LATEST_VERSION = SUPPORTED_VERSIONS[-1]
 
-__all__ = ['check_database', 'download_dbase', 'get_test_file_list', 'build_hdf5_dbase']
+__all__ = ['check_database', 'check_database_version', 'download_dbase', 'md5hash', 'get_test_file_list', 'build_hdf5_dbase']
```

Comment:
I think I should just move these functions to be private. I excluded from the top level namespace here for a reason.

Reply:
This can be separate PR.